multi-agent/lib/mas/resources/registry.py

```
@@ -15,9 +15,11 @@ def __init__(
             self,
             repo: ResourceRepository,
             bp_repo: BlueprintRepository,
+            cipher: Optional[Any] = None,
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `cipher`

(ANN401)